src/components/AboutUs/AboutUs.scss

```diff
@@ -0,0 +1,95 @@
+@import "../../styles/variables";
+
+.about__container {
+  margin-top: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.about__info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-basis: 50%;
+}
+
+.about__info_text {
+  font-weight: 600;
+  width: 515px;
+  margin-bottom: 15px;
+}
+
+.about__info__button {
+  display: flex;
+  justify-content: flex-end;
+  border: 2px solid $lightThemeAccept;
```

Comment:
Sort these  CSS properties by groups 
, so all related properties will be in one place. They should not be placed randomly inside selector specification

Reply:
Verify this rule for all CSS properties in this PR